### ISSUE TITLE
[AXE-CORE]: Outreach hub - Links must have discernible text

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#22138]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where empty link tags were on the pages. These are problematic because screen readers will announce "Link" but not what the link is. 

There are 4 errors in [this event](https://www.va.gov/outreach-and-events/events/accessing-covid-19-vaccinations-at-the-va-what-you-need-to-know/) because of empty `em` tags. The italics are also not consistent with the style guide. 

 
![image](https://user-images.githubusercontent.com/55411834/112393235-60c40280-8cc0-11eb-95b8-d82a113c61dd.png)

![image](https://user-images.githubusercontent.com/55411834/112393085-16db1c80-8cc0-11eb-812c-69a8eb91143f.png)

## Tasks
- [ ] If there is no value or HREF, the link `<a></a>` should be removed. 

## Acceptance Criteria
- [ ] All instances of the error are fixed
- [ ] Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

**Comment by mmiddaugh** [comment]
This item was pulled out of https://github.com/department-of-veterans-affairs/va.gov-team/issues/21791

**Comment by kelsonic** [comment]
Ah ha! I found it: `build/localhost/outreach-and-events/events/accessing-covid-19-vaccinations-at-the-va-what-you-need-to-know/index.html` This is a page that's built by the CMS so a content editor will need to fix this I think CC @RLHecht

**Comment by brianalloyd** [comment]
@RLHecht would you like to move this to your board for tracking? LMK

**Comment by RLHecht** [comment]
@brianalloyd It looks like this fix was already done in Drupal by someone else so we should be good. For future issues, Sitewide Content doesn't handle Outreach and Events. I reached out to the CMS team to find out which team does and can let you know when I hear back. I think it would be good for that to be aware of these issues that pop up so they can learn how to fix them. Seems like it's often how they are adding links to their pages. Thanks!

**Comment by brianalloyd** [comment]
@RLHecht thanks for the work on this and the context, it helps for sure.  I'll work with CMS forward on Outreach and Event hubs as sometimes finding who owns what on VSA is a challenge.  Thanks again.

**Comment by 1Copenut** [comment]
Verified this one is fixed. Moving to closed.
